Research/readme.md

```diff
@@ -26,23 +28,54 @@ From a terminal launch the run_docker_notebook.bat/.sh script; there are a few o
 
-Once the docker image starts, the script will attempt to open your browser to the Jupyter notebook web app, if this fails go to `localhost:8888`
+Once the docker image starts, the script will attempt to open your browser to the Jupyter notebook web app, if this fails open your browser and go to `localhost:8888`
 
-When you are done with the research environment be sure to stop the container with either Docker's dashboard or through the CLI.
+<br>
 
-## Note for C#
-When using C# for research notebooks it requires that you load our CSX file `QuantConnect.csx` into your notebook. In this setup, the file is one directory above the notebooks default dir. Be sure to use the following line to load in this csx file:
+## C# Notebook
+When using C# for research notebooks it requires that you load our setup script CSX file `QuantConnect.csx` into your notebook. This will load our QuantConnect libraries into your C# Kernel. In this setup, the file is one directory above the notebooks dir. Be sure to use the following line in your first cell to load in this csx file:
 
 `load "../QuantConnect.csx"`
 
+After this the environment is ready to use; take a look at our reference notebook `KitchenSinkCSharpQuantBookTemplate.ipynb` for an example of how to use our `QuantBook` interface!
+
+<br>
+
+## Python Notebook
+With Python we have a setup script that will automatically load QuantBooks libraries into the Python kernel so there is no need to import them. 
+
+You notebook is ready to use; take a look at our reference notebook `KitchenSinkQuantBookTemplate.ipynb` for an example of how to use our `QuantBook` interface!
+
+<br>
+
+## Using the Web Api from Notebook
+Both of our setup scripts for Python & C# include a instantiated `Api` object under the variable name `api`. Before you can use this api object to interact with the cloud you must edit your config in the **root** of your Notebook directory. Once this has been done once, it does not need to be done again.
+
+In `config.json` add the following entries with your respective values
+```
+job-user-id: 12345 // Your id here
+api-access-token: "token13432" // Your api token here
+```
+
+Once this has been done, you may restart your kernel and begin to use the `api` variable. 
+Reference our examples mentioned above for practical uses of this object.
+
+<br>
+
+## Shutting Down the Notebook Lab
+When you are done with the research environment be sure to stop the container with either **Docker's dashboard** or through the **Docker CLI**.
+
+<br>
+
+
 ## Build a new image
 For most users this will not be necessary, simply use `docker pull quantconnect/research` to get the latest image.
 
-
 `docker build -t quantconnect/research - < DockerfileJupyter` will build a new docker image using the latest version of lean. To build from particular tag of lean a build arg can be provided, for example `--build-arg LEAN_TAG=8631`.
 
+<br>
 
-# Running Jupyter Locally
+# Running Jupyter Locally (Not Recommended)
```

Comment:
Maybe instead of `(Not Recommended)` at the title, we could add a sentance like: `Note: we recommend the previous approach because compared with this one, the setup and environment is already tested and fixed in the docker image...etc`